### DISCUSSION TITLE
When will I get an answer to my ticket in support?

### DISCUSSION THREAD

**Discussion by PavelTurk** [community/community#139705] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

8 days ago I opened an issue in support but still got no answer. Could anyone say fast supports answers? To tell the truth I didn't expect that it would take more than 3 days.

**Comment by dotexe3301** [comment]
What was your issue?

**Comment by ebndev** [comment]
Hi @PavelTurk,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
